modules/claims_api/lib/bgs_service/power_of_attorney_request_service.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class PowerOfAttorneyRequestService < ClaimsApi::LocalBGS
+    PROC_TYPE_CD = 'POAAUTHZ'
+
+    def proc_create
```

Comment:
Another option might be to take the type cd as a param with the default set to `POAAUTHZ`

Reply:
we can loop back on that as a future update if needed. going to add this to the canvas as a question - if all 21-22 and 21-22a are `POAAUTHZ` or if we'll have other use cases.